innopoints/schemas/account.py

```diff
@@ -30,3 +32,15 @@ class Meta:
     entry_time = ma.DateTime(timezone=True)
     type = ma.Str()
     payload = ma.Dict()
+
+
+class NotificationSettingsSchema(ma.Schema):
+    class Meta:
+        ordered = True
+        sqla_session = db.session
+
+    innostore = ma.Str(validate=validate.OneOf(['off', 'email', 'push']))
+    volunteering = ma.Str(validate=validate.OneOf(['off', 'email', 'push']))
+    project_creation = ma.Str(validate=validate.OneOf(['off', 'email', 'push']))
+    administration = ma.Str(validate=validate.OneOf(['off', 'email', 'push']))
+    service = ma.Str(validate=validate.OneOf(['off', 'email', 'push']))
```

Comment:
`service` cannot be off
```suggestion
    service = ma.Str(validate=validate.OneOf(['email', 'push']))
```

Reply:
I told you about it, I don't think we really need special handling for this in the backend. Just prohibit it on the frontend and forget about it

Reply:
That's not really "special handling". Just remove it from the list (although now that the list is refactored, it would take more work, so ok, forget about it)